### DISCUSSION TITLE
Make lint warn/error paths as link

### DISCUSSION THREAD

**Discussion by sbsrnt** [biomejs/biome#1729] [Ideas]
Here is how it's looking currently in biome's lint
![image](https://github.com/biomejs/biome/assets/19862545/9fda1ff7-f54d-42cd-b393-b280a4bca1a9)

And here is the eslint equivalent in JetBrains' IDEs
![image](https://github.com/biomejs/biome/assets/19862545/0aaee044-51ae-41fb-834b-6dd4d77dc530)

---
During short conversation on discord, @strager [mentioned how `quick-lint-js` does linking for URLs](https://github.com/quick-lint/quick-lint-js/blob/434d08514d2ab0bc70b5ecb40f1e1e7c1256b6ae/src/quick-lint-js/cli/text-diag-reporter.cpp#L87-L92) indicating a potential similarity for file paths.

Personally I don't know how one would achieve this as my knowledge is limited in this area.
Maybe @segrey can share some ideas regarding how JetBrains handles this with `eslint`.

**Comment by ematipico** [comment]
It's a reasonable feature, although we would want to make it generic, not tailored around one single editor
